hasher-matcher-actioner/hmalib/fetching/bank_store.py

```diff
@@ -315,5 +316,5 @@ def convert_opinions_to_label(self, opinions: t.Sequence[SignalOpinion]):
         ddb using self.banks_table) can be written.
         """
         return reduce(
-            lambda tags, acc: acc.union(tags), map(lambda x: x.tags, opinions)
+            lambda tags, acc: acc.union(tags), map(lambda x: x.tags, opinions), []
```

Comment:
Errored out if opinions was empty. I'm sure there is a comprehension that would work, but that's a fight for another day.

Reply:
Strongly enough, the inner map is already basically a comprehension, just needs the map removed.

As written, I think this is going to create a lot of intermediary objects.

```
return {t for o in opinions for t in o.tag}
```
Since the return type is sequence, you may need wrap with `list()`